_articles/download-data.md

```diff
@@ -0,0 +1,96 @@
+
+Once you have identified the files you want to download from Explore Data, **Export Table** from Download Options.
+
+<img style="width: 25%;" src="/assets/images/export-table-viz.png" alt="alt text">
+
+You may choose to download the file as a *.csv* or *.tsv*. Files are named *Job-####*, where # includes a long set of numbers. Move this file to your working directory to proceed with the following steps. 
+
+[Install the Synapse R client](https://r-docs.synapse.org/#installation) `synapser` to download data from Synapse. [Login](https://r-docs.synapse.org/articles/manageSynapseCredentials.html#manage-synapse-credentials) to Synapse.
+
+```
+library(synapser)
+synLogin("my_username", "my_password")
+```
+Read the exported table into R, create a directory to store files and download data using `synGet`. If `downloadLocation` is not specified, the files are downloaded to a hidden directory called `/.synapseCache`. 
+```
+exported_table <- read.csv("Job-1342324092830598.csv")
+dir.create("files")
+lapply(exported_table$id, function(x) synGet(x,
+                                             downloadLocation = "./files"))
+```
+
+The `exported_table` also includes experimental details relevant to how the data was processed.
+
+### Python
+
+In order to download data programatically, you need a list of synIDs that correspond to the files.
+
+Once you have identified the files you want to download from Explore Data, **Export Table** from Download Options.
+
+<img style="width: 25%;" src="/assets/images/export-table-viz.png" alt="alt text">
+
+You may choose to download the file as a *.csv* or *.tsv*. Files are named *Job-####*, where # includes a long set of numbers. Move this file to your working directory to proceed with the following steps. 
+
+[Install the Synapse Python client](http://python-docs.synapse.org/build/html/index.html#installation) `synapseclient` to download data from Synapse, the `pandas` library to read a csv file and the `os` module to make a directory. [Login](http://python-docs.synapse.org/build/html/index.html#connecting-to-synapse) to Synapse.
+```
+import synapseclient, pandas, os
+syn = synapseclient.Synapse()
+
+syn.login('my_username', 'my_password')
+```
+Read the exported table into Python, create a directory to store files and download data using `syn.get`. If `downloadLocation` is not specified, the files are downloaded to a hidden directory called `/.synapseCache`.
+
+```
+exported_table = pandas.read_csv("Job-1342324092830598.csv")
+os.mkdir("files")
+[syn.get(x, downloadLocation = "./files") for x in exported_table.id]
+```
+
+The `exported_table` also includes experimental details relevant to how the data was processed.
```

Comment:
Same comment above about using the correct vocabulary when talking about what this file contains (i.e. annotations).